.github/workflows/test-upgrade.yaml

```diff
@@ -5,19 +5,19 @@ on:
   workflow_dispatch:
     inputs:
       kuadrantStartVersion:
-        description: Kuadrant start version
+        description: Kuadrant start version (format X.Y.Z)
```

Comment:
do we mean to upgrade from a `pre-version` tag?

Reply:
I do not see a use case where you would want to test upgrade _from_ a release candidate to something else. The common use case would be upgrade _from_ a existing release to some candidate release or existing release.